tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -0,0 +1,187 @@
+
+				// Wait for all XHRs and redirects.
+				await page.waitForNavigation( { waitUntil: 'networkidle0' } );
+
+				// Expect Jetpack call to be executed.
+				expect( jetpackConnectMock ).toBeCalled();
+				// Expect the user to be redirected
+				expect( page.url() ).toEqual( createURL( '/auth/url/' ) );
+			} );
+		} );
+	} );
+	describe( 'Merchant who has their Jetpack connected, but Google not yet connected', () => {
+		beforeEach( async function goToSetup() {
+			// Mock Jetpack as connected
+			await requestMock.observe( page );
+			requestMock
+				.mock( /%2Fwc%2Fgla%2Fjetpack%2Fconnected\b/ )
+				.mockImplementation( ( interceptedRequest ) => {
+					interceptedRequest.respond( {
+						content: 'application/json',
+						headers: { 'Access-Control-Allow-Origin': '*' },
+						body: JSON.stringify( {
+							active: 'yes',
+							owner: 'yes',
+							displayName: 'testUser',
+							email: 'mail@example.com',
+						} ),
+					} );
+					return true;
+				} );
+
+			// Mock google as not connected.
+			// When pending even WPORG will not render yet.
+			// If not mocked will fail and render nothing,
+			// as Jetpack is mocked only on the client-side.
+			requestMock
+				.mock( /%2Fwc%2Fgla%2Fgoogle%2Fconnected\b/ )
+				.mockImplementation( ( interceptedRequest ) => {
+					interceptedRequest.respond( {
+						content: 'application/json',
+						headers: { 'Access-Control-Allow-Origin': '*' },
+						body: JSON.stringify( {
+							active: 'no',
+							email: '',
+						} ),
+					} );
+
+					return true;
+				} );
+
+			// Go to the setup page short way - directly via URL.
+			await visitGLAPage( { path: '/google/setup-mc' } );
+			// Wait for API calls and the page to render.
+			await page.waitForXPath( "//*[text()='Set up your accounts']" );
+		} );
+
+		it( 'should see their WPORG email, "Connect your Google account" text & connect button', async () => {
+			await expect(
+				page.waitForXPath( "//*[text()='mail@example.com']" )
+			).resolves.toBeTruthy();
+
+			await expect(
+				page.waitForXPath( "//*[text()='Connect your Google account']" )
+			).resolves.toBeTruthy();
+
+			await expect(
+				page.$x( "//button[text()='Connect'][not(@disabled)]" )
+			).resolves.toBeTruthy();
+		} );
+
+		describe( 'after clicking the "Connect your Google account" button', () => {
+			let googleConnectMock;
+			beforeEach( async function mockJetpackConnect() {
+				// Spy on Google connection request.
+				googleConnectMock = requestMock
+					.mock( /%2Fwc%2Fgla%2Fgoogle%2Fconnect\b/ )
+					.mockImplementation( ( interceptedRequest ) => {
+						interceptedRequest.respond( {
+							content: 'application/json',
+							headers: { 'Access-Control-Allow-Origin': '*' },
+							body: JSON.stringify( {
+								url: createURL( '/google/auth/' ),
+							} ),
+						} );
+						return true;
+					} );
+			} );
+
+			it( 'should sent an API request to connect Jetpack, and redirect to the returned URL', async () => {
```

Comment:
💅 
The `"Jetpack"` looks like missed replacing to a text such as "Google auth".